static/entry.js

```diff
@@ -35,4 +35,9 @@ import ManualInitializer from './js/manual-initializer';
 export { ManualInitializer };
 
 import RuntimeConfigReceiver from './js/runtime-config-receiver';
-export { RuntimeConfigReceiver };
\ No newline at end of file
+export { RuntimeConfigReceiver };
+
+import RuntimeConfig from './js/runtime-config';
+window.AnswersExperience = {
```

Comment:
If we have this here, do we need the `window.AnswersExperience = {};` in `layouts/html.hbs`?

Reply:
nope. removing